projectTemplates/ts-server/src/middleware/ErrorHandler.ts

```diff
@@ -15,10 +17,14 @@ export class ErrorHandler {
       // eslint-disable-next-line @typescript-eslint/no-unused-vars
       next: NextFunction
     ): void => {
-      this.logger.error(
-        `${req.method} request to ${req.originalUrl}  has failed with error: ${err.message}`
-      );
-      res.sendStatus(StatusCodes.INTERNAL_SERVER_ERROR);
+      if (err instanceof InputValidationError) {
+        res.status(HttpStatus.BAD_REQUEST).json(err.errors);
+      } else {
+        this.logger.error(
+          `${req.method} request to ${req.originalUrl}  has failed with error: ${err.message}`
+        );
+        res.sendStatus(StatusCodes.INTERNAL_SERVER_ERROR);
+      }
```

Comment:
depends on NODE_ENV 

if NODE_ENV is production then we want to return status of 500 
but if it NODE_DEV=development we would like to output thee message and stacktrace

Reply:
by default the NODE_ENV seams empty.
do we want to add it to .env.example?
or do we want to check if it is not "production"? seams more risky but is set in dockerfile